src/components/UI/Form/FormItem/FormItem.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { TextField } from '@material-ui/core';
+
+const FormItem = (props) => {
```

Comment:
Изменить на деструктуризацию
```suggestion
const FormItem = () => {
```

Reply:
изменено на деструктуризацию